crates/compiler/gen_dev/src/generic64/aarch64.rs

```diff
@@ -3953,6 +3973,24 @@ fn fsub_freg_freg_freg(
     buf.extend(inst.bytes());
 }
 
+/// `FNEG Sd/Dd, Sn/Dn`
+#[inline(always)]
+fn fneg_freg_freg(
```

Comment:
I'm so sorry that our naming scheme has lead to this... haha

Reply:
in my head, I pronounce `freg` as "eff rej" but `fneg` as "eff neg"